src/extension.ts

```diff
@@ -16,7 +16,8 @@ import {
 } from "./env0-environments-provider";
 import { getEnvironmentsForBranch } from "./get-environments";
 
-export const ENV0_BASE_URL = "api-dev.dev.env0.com";
+export const ENV0_BASE_URL = process.env.ENV0_BASE_URL || "api.env0.com";
```

Comment:
If the env variable is not set use the production URL.
Tested locally with the dev api.